spec/model_spec.cr

```diff
@@ -1,4 +1,5 @@
 require "./spec_helper"
+require "uuid"
```

Comment:
I think this should be moved to `src/lucky_record.cr` so the requires are (mostly) in one place!